__mocks__/handlers/task-details.handler.ts

```diff
@@ -14,17 +14,18 @@ const taskDetailsHandler = [
                         dinero: 0,
                         neelam: 0,
                     },
-                    purpose: 'string',
-                    endsOn: 1618790400,
+                    purpose: 'This is a sample description',
+                    endsOn: 1618790410,
```

Comment:
can we change and use `Date` object, so that we know from looking at the code what value it is set on?

Reply:
There is a dedicated function `covertToTimestamp` to convert this time stamp and show it on the page.